requirements.txt

```diff
@@ -1,4 +1,4 @@
-pandas
+pandas>=1.0.0
```

Comment:
Do we need greater than 1? I thought the issues were with pandas<0.25?

Reply:
The pandas versions jumped from v0.25 to v1.0, so v1.0 is the first version that will work.